projects/valtimo/dossier/src/lib/components/dossier-detail/dossier-detail.component.html

```diff
@@ -182,6 +182,17 @@
   </cds-overflow-menu-option>
 </ng-template>
 
+<ng-template #deleteButton let-canDelete="canDelete">
+  <cds-overflow-menu-option
+          *ngIf="canDelete"
+          [disabled]="(isDeleting$ | async)"
+          (click)="deleteDocument()"
+          type="danger"
+  >
+    <span>{{ 'dossier.delete.delete' | translate }}</span>
```

Comment:
Do you need the <span>?

Reply:
Which '?' are you talking about? i don't see any in this block.

Reply:
It was meant for the <span>, I suppose I forgot to type it

Reply:
Removed the span and the other ones in similar spots to be consistent. Seems to work fine without.